core/math/math_funcs.h

```diff
@@ -317,29 +317,14 @@ class Math {
 		return u.d;
 	}
 
-	//this function should be as fast as possible and rounding mode should not matter
-	static _ALWAYS_INLINE_ int fast_ftoi(float a) {
+	static _ALWAYS_INLINE_ int fast_ftoi(const float &a) {
```

Comment:
What's the point of adding a method that nothing use and then have to rename all occurrences of the original method? Seems kind of pointless.

Reply:
Sorry I am having trouble with Git, I have something like this now :

```
static _ALWAYS_INLINE_ int fast_ftoi(const float &a);
static _ALWAYS_INLINE_ int fast_ftoi_val(const float a);
```

Simply because half of the usages could have been pass by reference.

Reply:
But what problem are you trying to solve? It sounds like you're trying to optimize something without really checking if it actually is an optimization.

Reply:
Hmm, you are right. I will try to profile this.